src/components/welcome/welcome.scss

```diff
@@ -1,9 +1,8 @@
 .welcome-image-mask {
   position: relative;
-  height: 420px;
+  height: 380px;
   overflow: hidden;
   width: 100%;
-  max-width:1170px;
```

Comment:
Here's what it looks like at 2800px width: 
![image](https://cloud.githubusercontent.com/assets/31034/16117882/a2f8759a-33d4-11e6-8800-1144830d526b.png)
The content keeps aligned with the repos box container.